Main/ClientApp/src/Components/Drawer/MyCollectionsSection.tsx

```diff
@@ -65,17 +98,59 @@ export default function MyCollectionsSection() {
                 icon={collection.isPublic ? <Public /> : <Lock />}
                 link={`/collections/${collection.id}`}
                 menuItems={
-                  //<></>}
-                  /*
-                  // @ts-ignore */
-                  <ListItem
-                    onClick={() => onShareCollectionClick(collection.id)}
-                  >
-                    <ListItemIcon>
-                      <GroupAdd />
-                    </ListItemIcon>
-                    <ListItemText primary="Share" />
-                  </ListItem>
+                  <>
+                    <Typography
+                      variant="h5"
+                      className={classes.collectionSettingsMenuTitle}
+                    >
+                      Settings
+                    </Typography>
+                    <MenuItem
+                      onClick={() => onDeleteCollectionClick(collection.id)}
+                    >
```

Comment:
Again, have you checked the console? I am pretty sure we can't have a listitem inside of menu item.